phpcs.xml

```diff
@@ -19,4 +19,5 @@
 	<exclude-pattern>*/dev-lib/*</exclude-pattern>
 	<exclude-pattern>*/node_modules/*</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
+	<exclude-pattern>*/includes/sanitizers/class-amp-allowed-tags-generated.php</exclude-pattern>
```

Comment:
This file is generated by the Python file, and created hundreds of warnings and errors in PHPCS.